templates/app/assets/stylesheets/spree/frontend/components/cart/_cart_link.scss

```diff
@@ -1,19 +1,33 @@
 .cart-link {
```

Comment:
@mfrecchiami I copied the CSS from Solidus demo. Let me know if you find any issues with the CSS.